core/state/statedb.go

```diff
@@ -256,6 +256,32 @@ func (self *StateDB) GetState(addr common.Address, hash common.Hash) common.Hash
 	return common.Hash{}
 }
 
+// MerkleProof
+type ProofList [][]byte
+
+func (n *ProofList) Put(key []byte, value []byte) error {
+	*n = append(*n, value)
+	return nil
+}
+
+// returns the MerkleProof for a given Account
+func (self *StateDB) GetProof(a common.Address) [][]byte {
+	var proof ProofList
+	self.trie.Prove(crypto.Keccak256(a.Bytes()), 0, &proof)
```

Comment:
`trie.Prove` actually returns an `error`, which you're ignoring here. Wouldn't it be better to bubble this up, and return `([][]byte, error)`?

Reply:
good point.... Errors are now handled correctly.